### PR TITLE
Dg photogram industrial 2 Pull request

### PR DESCRIPTION
**Pull request by dgarcia13-99**

### CODE REVIEW

**Review by hatcheta1 — APPROVED** [review]
looks good!

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#  index_comments_on_photo_id  (photo_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (author_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: "User"
+  belongs_to :photo
```
Comment (JoseMDO) [inline app/models/comment.rb]: Great associations!

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
+#  index_users_on_username              (username) UNIQUE
+#
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :own_photos, class_name: "Photo", foreign_key: "owner_id"
```
Comment (JoseMDO) [inline app/models/user.rb:32]: Nice job!

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
+  resources :likes
```
Comment (JoseMDO) [inline config/routes.rb]: Routes all look great!

File: app/controllers/likes_controller.rb [hunk 4]
```diff
@@ -0,0 +1,70 @@
+class LikesController < ApplicationController
```
Comment (JoseMDO) [inline app/controllers/likes_controller.rb:1]: Controllers all look great! Great job using scaffolding to create your rcav's :)

**Review by JoseMDO — APPROVED** [review]
Awesome job Dani!

**Pull request merged**